server/client.go

```diff
@@ -210,7 +210,7 @@ type client struct {
 	nonce      []byte
 	pubKey     string
 	nc         net.Conn
-	ncs        string
+	ncs        atomic.Value
```

Comment:
Move this to line 191 (above cgwrt), for the reason explained in the comment just before cgwrt.

Reply:
Actually, scratch that since `atomic.Value` is itself a struct, and Load()/Store() will work on the inside value of `v interface{}` which is aligned.